requirements.txt

```diff
@@ -0,0 +1,5 @@
+ConfigObj
+files
+pillow
+pymsgbox
```

Comment:
If we opt to use the tkinter standard messagebox, we may not need to include pymsgbox as this library is no longer used.